main/src/main/java/org/mobilitydata/gtfsvalidator/notice/SameNameAndDescriptionForRouteNotice.java

```diff
@@ -20,6 +20,12 @@
 
 import com.google.common.collect.ImmutableMap;
 
+/** A {@code GtfsRoute} has identical value for `routes.route_desc` and
+ * `routes.route_long_name`/`routes/route_short_name.
+ * <p>"Do not simply duplicate the name of the location." (http://gtfs.org/reference/static#routestxt)
+ *
+ * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
This should be a WARNING. This does not make the feed non-functional.

Reply:
> I think we can safely call this an ERROR. The spec says do not.

from https://github.com/MobilityData/gtfs-validator/pull/726#discussion_r574703383